pages/docs/index.tsx

```diff
@@ -96,24 +107,39 @@ const Docs = ({ bitbucket_auth_url, image_name }: { bitbucket_auth_url: string,
 							<ProviderSelector providerOptions={providerOptions} selectedProvider={selectedProvider} setSelectedProvider={setSelectedProvider} />
 						</label>
 						<label className='flex justify-between font-semibold text-sm'>Installation Type:
-							<InstallationSelector selectedInstallation={selectedInstallation} setSelectedInstallation={setSelectedInstallation}/>
+							<InstallationSelector selectedInstallation={selectedInstallation} setSelectedInstallation={setSelectedInstallation} />
 						</label>
 						<label className='font-semibold text-sm w-full flex justify-between'>Hosting:
-							<HostingSelector selectedHosting={selectedHosting} setSelectedHosting={setSelectedHosting}/>
+							<HostingSelector selectedHosting={selectedHosting} setSelectedHosting={setSelectedHosting} />
 						</label>
 					</AccordionContent>
 				</AccordionItem>
 				<AccordionItem value="instruction-3" disabled={selectedHosting === '' || !selectedProvider || selectedInstallation === ''}>
 					<AccordionTrigger>Set up DPU</AccordionTrigger>
 					<AccordionContent>
-						<BuildInstruction selectedHosting={selectedHosting} userId={getAuthUserId(session)} selectedInstallationType={selectedInstallation} selectedProvider={selectedProvider!!} session={session} />
+						<BuildInstruction selectedHosting={selectedHosting} userId={getAuthUserId(session)} selectedInstallationType={selectedInstallation} selectedProvider={selectedProvider!!} session={session} installId={installId} />
 					</AccordionContent>
 				</AccordionItem>
 				<AccordionItem value="instruction-4" disabled={!selectedProvider}>
-					<AccordionTrigger>Set up triggers</AccordionTrigger>
-					<AccordionContent>
-						<TriggerContent selectedProvider={selectedProvider} bitbucket_auth_url={bitbucket_auth_url} selectedHosting={selectedHosting} selectedInstallationType={selectedInstallation}/>
-					</AccordionContent>
+					{selectedProvider === 'github' && selectedHosting === 'selfhosting' && selectedInstallation === 'individual' ? (
+						<>
+							<AccordionTrigger>Repository Selection</AccordionTrigger>
+							<AccordionContent>
+								{installId ? (
+									<RepoSelection repoProvider={selectedProvider as RepoProvider} installId={installId as string} setIsRepoSelectionDone={null} />
+								) : (
+									<>User Info not found, please refresh and try again.</>
+								)}
```

Comment:
Why did you not copy the `isGetInstallIdLoading` state as well?

Here, the user will see the failure message even when the API call to get the install id hasn't failed yet. And they might refresh the page, triggering the API again and not waiting until the API call completes.

Reply:
Got it, missed this for some reason. Will create an issue and a new PR for this